src/BPEssentials/Commands/Home/Home.cs

```diff
@@ -0,0 +1,40 @@
+﻿using BPEssentials.Abstractions;
+using BPEssentials.ExtensionMethods;
+using BrokeProtocol.Entities;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace BPEssentials.Commands
+{
+    public class Home : Command
+    {
+        public void Invoke(ShPlayer player, int homeNumber=1)
+        {
+
+            List<ShApartment> apartments = new List<ShApartment>(player.ownedApartments.Keys);
+            if (apartments.Count == 0)
+            {
+                player.TS("No_Appartments");
+                return;
+            }
+            else if (apartments.Count < homeNumber || homeNumber < 1)
```

Comment:
To make this more readable, switch the two sides around.
```suggestion
            if (homeNumber > apartments.Count)
```